src/middlewared/middlewared/plugins/security/update.py

```diff
@@ -44,6 +44,9 @@ async def do_update(self, data):
                 'Please contact iX sales for more information.'
             )
 
+        if await self.middleware.call('failover.disabled.reasons'):
```

Comment:
We need to allow someone the ability to enable FIPS and then subsequently disable it without having to reboot and failover.